v3/docs/gs/sysreqs.md

```diff
@@ -24,12 +24,11 @@ be helpful in creating and verifying a balanced configuration.
 ## Operating System
 
 HSE should work with most modern Linux&reg; 64-bit operating system
-distributions.  We have run HSE on the following.
+distributions.  We have run HSE on the following:
 
-* Red Hat&reg; Enterprise Linux&reg; 8 (RHEL 8)
-* Ubuntu&reg; 18.04 and 20.04
-* Fedora&reg; 34 and 35
-* AlmaLinux OS 8
+* Red Hat&reg; Enterprise Linux&reg; 8 (RHEL 8) and 9 (RHEL 9)
```

Comment:
```suggestion
* Red Hat&reg; Enterprise Linux&reg; (RHEL) 8 and 9
```

Reply:
Red Hat is not completely consistent in precisely where they introduce the acronym -- I've seen it both ways in the docs.  I have no preference and am happy to change it to the other.

Reply:
I think the suggestion will save the needless repetition of 8 and 9